test/end-to-end/vehicle.js

```diff
@@ -63,6 +71,6 @@ test('vehicle has permissions', async(t) => {
     ['read_odometer', 'read_vehicle_info']));
 });
 
-test('vehicle disconnect', async(t) => {
+test.after.always('vehicle disconnect', async(t) => {
```

Comment:
Nothing against using test.after, wondering though if you could explain why this was not necessary before?

Reply:
We need it because of a race condition that was happening where `vehicle disconnect` would run before some other tests had finished, which wiped out our permissions, and it would randomly cause certain random permissions to register as `false` on every test run. So the question is really why _wasn't_ that race condition happening before, and my best guess is that it's because the number of tests increases the likelihood of that race condition happening.